website/docs/r/cognito_user_pool.markdown

```diff
@@ -86,10 +86,10 @@ The following arguments are supported:
 
 #### Schema Attributes
 
-  * `attribute_data_type` (Optional) - The attribute data type.
+  * `attribute_data_type` (Required) - The attribute data type. Must be one of `Boolean`, `Number`, `String`, `DateTime`.
```

Comment:
Since we put that as Required (which is a good catch by the way), we should also change the related code attribute from Optional to Required

Reply:
Ah. Thanks. I was thinking I was just helping out with the documentation, but of course you're right. Code updated as well.